test/extended/images/mongodb_ephemeral.go

```
@@ -24,8 +24,13 @@ var _ = g.Describe("[images][mongodb] openshift mongodb image", func() {
 			g.By("creating a new app")
 			o.Expect(oc.Run("new-app").Args("-f", templatePath).Execute()).Should(o.Succeed())
 
-			g.By("expecting the mongodb service get endpoints")
-			o.Expect(oc.KubeFramework().WaitForAnEndpoint("mongodb")).Should(o.Succeed())
+			g.By("waiting for the deployment to complete")
+			exutil.WaitForADeployment(
```

Silently ignoring the error from `WaitForADeployment` :crying_cat_face: